app/routes/synonyms/synonyms.lib.ts

```diff
@@ -5,15 +5,23 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+import type { Circular } from '~/routes/circulars/circulars.lib'
 
 /* Data structure in DynamoDB */
 export interface Synonym {
   eventId: string
   synonymId: string
+  slug: string
 }
 
 /* Layout of materialized view in OpenSearch */
 export interface SynonymGroup {
   synonymId: string
   eventIds: string[]
+  slugs: string[]
+}
+
+export interface SynonymGroupWithMembers {
```

Comment:
Might be a little cleaner to extend SynonymGroup rather than have it as a member.